### DISCUSSION TITLE
Unable to create user packages in NetBeans

### DISCUSSION THREAD

**Discussion by abhich260** [apache/netbeans#7947] [General]
I am unable to create user packages or packages with kind of customized name from user that is my end except for the already created packages by default like Source packages or default package in Netbeans after I am creating a new java project . Everytime I am attempting to create Netbeans is creating a file with the name i am entering for package instead of creating package . The NetBeans version is 9.0 and I am launching the netbeans ide application by extracting the incubating Netbenas bin compressed folder that is available from apache net beans site becasue installer zip folder is not available for installing net beans 9.0 . 
Anyone Please help , provide solution for this issue .

**Comment by neilcsmith-net** [comment]
Only NetBeans 23 is supported. Please upgrade, try that, and see if you still see the same issue.
